packages/safe-ds-lang/tests/language/generation/safe-ds-python-generator.test.ts

```diff
@@ -36,6 +36,7 @@ describe('generation', async () => {
                     destination: test.actualOutputRoot,
                     createSourceMaps: true,
                     targetPlaceholder: runUntilPlaceholderName,
+                    disableRunnerIntegration: test.testName.startsWith('[eject'), // Tests in the "eject" suite are tested with disabled runner integration
```

Comment:
The `creator.ts` files are meant to create full descriptions of the tests. The `xyz.test.ts` then simply run those tests. In light of this separation of concerns, I'd suggest updating the `generation/creator.ts` file instead:

* Add a `disableRunnerIntegration` property to the `GenerationTest` interface.
* Populate the property in `createGenerationTest` by checking the file name.
* In `safe-ds-python-generator.test.ts`, simply pass the value of this property along.

Reply:
Also, update `docs/development/generation-testing.md`, please.